### ISSUE TITLE
Hard to figure out that a gene is only annotated in RefSeq

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#952]
I had a user question, and as I'm trying to use beta as much as possible, I did the search there.  The user said that a gene (QI18_03300)  lacked PLFams.  In production, the gene page looks like this:
<img width="1002" alt="screen shot 2016-09-29 at 4 58 41 pm" src="https://cloud.githubusercontent.com/assets/11475316/18972290/871a8a40-8666-11e6-83f6-718d8263dc64.png">

Its clear to me that the patric locus tag is missing.  In beta it looks like this:
<img width="1057" alt="screen shot 2016-09-29 at 4 59 38 pm" src="https://cloud.githubusercontent.com/assets/11475316/18972305/9727bd18-8666-11e6-922c-ed3c2030dc29.png">

Not as clear

**Comment by olsonanl** [comment]
BTW even with the latest families this protein gets assigned no family membership (it gets a single kmer hit for “ABC1 family protein”). Looks like something not currently annotated; When I blast against coreseed I don't get a hit for this particular protein.

When I ran the genome through the PATRIC annotation service Prodigal did call that protein and it properly propagated the locus tag but it called it a hypothetical protein.

On Sep 29, 2016, at 4:03 PM, ARWattam <notifications@github.com<mailto:notifications@github.com>> wrote:

I had a user question, and as I'm trying to use beta as much as possible, I did the search there. The user said that a gene (QI18_03300) lacked PLFams. In production, the gene page looks like this:
[screen shot 2016-09-29 at 4 58 41 pm]https://cloud.githubusercontent.com/assets/11475316/18972290/871a8a40-8666-11e6-83f6-718d8263dc64.png

Its clear to me that the patric locus tag is missing. In beta it looks like this:
[screen shot 2016-09-29 at 4 59 38 pm]https://cloud.githubusercontent.com/assets/11475316/18972305/9727bd18-8666-11e6-922c-ed3c2030dc29.png

Not as clear
